plugins/module_utils/helm.py

```diff
@@ -184,10 +184,10 @@ def get_helm_binary(self):
     def get_helm_version(self):
         command = self.get_helm_binary() + " version"
         rc, out, err = self.run_command(command)
-        m = re.match(r'version.BuildInfo{Version:"v([0-9\.]*)",', out)
+        m = re.match(r'version.BuildInfo{Version:"v([^"]*)",', out)
```

Comment:
I'd suggest getting rid of the character set and using a non-greedy operator:
```suggestion
        m = re.match(r'version.BuildInfo{Version:"v(.*?)",', out)
```

Reply:
Done.